internal/backup.go

```diff
@@ -34,7 +34,7 @@ type Backup struct {
 }
 
 func NewBackup(folder StorageFolder, name string) *Backup {
-	return &Backup{folder, name}
+	return &Backup{folder.GetSubFolder(BaseBackupPath), name}
```

Comment:
It doesn't work for some backup creation scenarios. E. g. in HandleDelete. Check for this change correctness in all usages of NewBackup function.

Reply:
Done.